tests/integration/specs/spec.dummy.js

```diff
@@ -1,6 +1,7 @@
-/*!
- * @author:    Divio AG
- * @copyright: http://www.divio.ch
+/*
+ * Copyright (c) 2013, Divio AG
```

Comment:
the copyright statement stands for the year it has been licensed to, which is 2015